cyral/resource_cyral_sidecar_listener.go

```diff
@@ -14,25 +14,36 @@ import (
 // create a constant block for schema keys
 
 const (
-	RepoTypesKey        = "repo_types"
-	NetworkAddressKey   = "network_address"
-	MySQLSettingsKey    = "mysql_settings"
-	DbVersionKey        = "db_version"
-	CharacterSetKey     = "character_set"
-	S3SettingsKey       = "s3_settings"
-	ProxyModeKey        = "proxy_mode"
-	DynamoDbSettingsKey = "dynamodb_settings"
+	RepoTypesKey         = "repo_types"
+	NetworkAddressKey    = "network_address"
+	MySQLSettingsKey     = "mysql_settings"
+	DbVersionKey         = "db_version"
+	CharacterSetKey      = "character_set"
+	S3SettingsKey        = "s3_settings"
+	ProxyModeKey         = "proxy_mode"
+	DynamoDbSettingsKey  = "dynamodb_settings"
+	SQLServerSettingsKey = "sqlserver_settings"
+	VersionKey           = "version"
```

Comment:
How about (for clarity)?

```suggestion
	SQLServerVersionKey           = "version"
```

Reply:
yes, I took your suggestion. Thanks